lib/OpenAPI/Client/OpenAI.pm

```diff
@@ -46,10 +65,11 @@ sub new {
 # install snake case aliases
 
 {
+    # Do we want to deprecate these? They're kind of a pain to maintain,
+    # or we should autogenerate them from the OpenAPI spec.
```

Comment:
i would recommend deprecating the aliases, b/c keeping the same name that's searchable in the API document is a win

Reply:
These are now deprecated in this PR.